doc/crud.md

```diff
@@ -485,17 +713,34 @@ document:
     CDTDocumentRevision *updated = [datastore createDocumentFromRevision:rev
                                                                    error:&error];
     ```
+    ```swift
+    let rev = CDTDocumentRevision("doc2")
+    rev.body = saved.body
+    rev.body["hello"] = "world"
+    // Create att100 attachment
+    rev.attachments = saved.attachments
```

Comment:
Is the `attachments` array a `copy` propery? Otherwise we're changing the original array here :/